docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       retries: 5
 
   plasma-deployer:
-    image: omisego/plasma-deployer:c7266b7
+    image: omisego/plasma-deployer:ff92de7
```

Comment:
No need to update since https://github.com/omisego/elixir-omg/pull/1054 already updated it to an even newer version